docs/docs/getting-started/glossary.md

```diff
@@ -0,0 +1,72 @@
+# Glossary
+
+### Contestation Period
+The time window between closing the head and `fan out` allowed to `contest`.
```

Comment:
This is a bit unclear 🤔

Reply:
What about: "The time window between closing of the head and the last possible contest from a head participant."